Translation corrections (#423)
07th-mod/higurashi-console-arcs: Update/ura_005.txt

			NULL, " Honestly, I was skeptical at first.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_okabe1023", 256, TRUE);
	OutputLine(NULL, "マニュアルを渡されたはいいが、よくもまぁいい加減なことを書いているものだ、なんて思いながら従っていましたよ」",

> 			NULL, " Giving me a manual's all fine and good, but I thought it was a bunch of crap while I was using it.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Giving me a manual's all well and good, but I thought it was a bunch of crap while I was using it.\"", Line_Normal);
```

//rそれを長年の捜査データをもとに体系づけたのが、茨城県警捜査一課長の本条。彼の論文は岡部も以前拝読して、…その詳細な内容に目をむいたものだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それを長年の捜査データをもとに体系づけたのが、茨城県警捜査一課長の本条。彼の論文は岡部も以前拝読して、…その詳細な内容に目をむいたものだ。",

> 			NULL, "This material was organized by the section chief of the Ibaraki prefectural police, based on long-term investigation data. Okabe had read the dissertation before... with those finer details opening his eyes.", Line_Normal);

Reviewer: ```suggestion
			NULL, "This material was organized by the section chief of the Ibaraki prefectural police, based on long-term investigation data. Okabe had read the dissertation before... with a fine eye on the details.", Line_Normal);
```

	ClearMessage();

//rそして、それらを激務の合間にレポートとして丁寧にまとめて、上層部の目を引くほどの理論的なマニュアルをくみ上げたのは、間違いなくあの『小娘』の努力の成果だった…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "そして、それらを激務の合間にレポートとして丁寧にまとめて、上層部の目を引くほどの理論的なマニュアルをくみ上げたのは、間違いなくあの『小娘』の努力の成果だった…。",

> 			NULL, "And it was undoubtedly the result of that \"little girl's\" efforts in-between her hard work of carefully compiling this as a report to create a theoretical manual that caught the eye of upper management...", Line_Normal);

Reviewer: ```suggestion
			NULL, "And it was carefully summarized even while in the midst of conducting hard work. It was undoubtedly a result of that \"little girl\" putting in the effort to create a theoretical manual that caught the eye of upper management...", Line_Normal);
```
